data/capture_settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "brightness": 115,
+  "contrast": 100,
+  "saturation": 185,
+  "temperature": 5280,
+  "exposure": 230,
+  "gain": 138
+}
```

Comment:
Just curious, are we going to always save the capture setting in here? or is it just a placeholder?

Reply:
I just found this https://github.com/ichiro-its/configuration. It's for ros2-ws robot's configuration. I will continue saving all configuration in this repo soon.